### PR TITLE
Doc types templates

### PR DESCRIPTION
**Pull request by lzdanski**
Add standard templates for doc types.

### CODE REVIEW

File: doc_type_templates/cli-reference.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+id: cli-ref-title
+description: Short cli description.
+hide_table_of_contents: true
+sidebar_custom_props: { icon: "img/term-icon.png" }
+---
+
+<!-- All CLI ref must include the `sidebar_custom_props` line in the header to ensure generated sidebar info renders correctly.-->
+<!-- See cli/astro-workspace-user-list.md for a command reference example that includes both software and astro commands, outputs, usage examples, and input options -->
+
+- Intro summary naming and defining the category, command, or feature
```
Comment (lzdanski) [inline doc_type_templates/cli-reference.md:13]: Describe what the command does or does not do. Include a link to relevant Docs pages. For example:

`astro workspace update` allows you to update the workspace metadata, including the workspace name and ci-cd deployment enforcement. It does not include adding  new users or changing permissions.  See [Configure workspaces](https://docs.astronomer.io/astro/manage-workspaces).

**Comment by lzdanski** [comment]
@jwitz - I finished up feedback from our meeting this morning. Ready for your review.

File: README.md [hunk 2]
```diff
@@ -17,4 +17,6 @@ GitHub issues are triaged by the Astronomer team and addressed promptly. Once yo
 
 ## Contribute
 
-See [CONTRIBUTING.MD](https://github.com/astronomer/docs/blob/main/contributing.md) for guidance on how to contribute to Astronomer documentation.
\ No newline at end of file
+See [CONTRIBUTING.MD](https://github.com/astronomer/docs/blob/main/contributing.md) for guidance on how to contribute to Astronomer documentation.
+
+You can also review the [Doc templates](https://github.com/astronomer/docs/blob/main/doc-templates) to see our standards for how to structure the content you want to contribute.
```
Comment (jwitz) [inline README.md:22]:
```suggestion
Do you have an idea for a new doc to contribute, but don't know where to start? Review the [Doc templates](https://github.com/astronomer/docs/blob/main/doc-templates) to see our standards for how to structure the content you want to contribute.
```

File: contributing.md [hunk 3]
```diff
@@ -31,6 +31,21 @@ The `software` directory is equivalent to the **Latest** version of the Astronom
 
 If you're working on a change in Software docs, you should work primarily in `software`. Make changes to `software_versioned_docs` only if your change is version-specific or a critical fix (e.g. incorrect or out-of-date information).
 
+### Doc templates
+
+If you want to contribute new docs to the Astro, Software, or Astro CLI docs, you can get started by copying one of the standard [Doc templates](https://github.com/astronomer/docs/blob/main/doc-templates) when you make your initial PR. These templates include links to examples of similar docs as well as guiding questions that your new content should answer.
+
+There are four standard templates:
+
+* **Overview** -  Gives a detailed explanation about what something is, how it functions as a part of Astronomer products or in an implementation, and why someone should use it. All docs should have at least one sentence explaining why a feature is important to a user.
```
Comment (jwitz) [inline contributing.md:40]: Is it worth it to hyperlink the names here to the relevant templates?

File: doc-templates/process.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+## Prerequisites
+- list any steps, other procedures, or required installations that a user must complete before attempting this procedure
+
+## Step 1:
+
+- If there are insufficent sections to have steps as sub-headings, consider using a different ## header like `## Setup` instead of `## Step 1:`
+
+## Step 2:
+...
```
Comment (jwitz) [inline doc-templates/process.md:22]:
```suggestion
You can nest steps within topics in the following way:

1. Substep 1
2. Substep 2
```

File: doc-templates/process.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+## Step Final:
```
Comment (jwitz) [inline doc-templates/process.md:24]:
```suggestion
## Step [Final]:
```

File: contributing.md (hunk 3 above)
Comment (lzdanski) [reply contributing.md:40]: I'll follow up with those after merging! Right now they don't exist in the main branch so it's a little tricky/ guessing

**Pull request merged**